### PR TITLE
Add date and time in the URL

### PR DESCRIPTION
**Pull request by vbochetRTE**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/utils/date-time-utils.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    const daysToIncrement = Number.isInteger(dayIncrement) ? dayIncrement : 0;
+    return setTimestampWithDaysIncrement(TODAY_TIMESTAMP, daysToIncrement);
+}
+
+function twoDigits(value) {
+    return ('0' + value).slice(-2);
+}
+
+function twoDigitsMonth(value) {
```
Comment (TheoPascoli) [inline src/utils/date-time-utils.js:58]: I dont understand what those two functions are doing, Why the +1 ? Maybe change the name or add comments

File: src/utils/view-utils.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export const RUNNING_TASKS_VIEW = 2;
+
+export function getInitialViewToSet(dateParam, timeParam, defaultView) {
+    if (dateParam?.match(DATE_REGEX)) {
+        if (timeParam?.match(TIME_REGEX)) {
+            return PROCESS_TIMESTAMP_VIEW;
+        }
+        return BUSINESS_DATE_VIEW;
+    }
+    return defaultView;
```
Comment (TheoPascoli) [inline src/utils/view-utils.js:21]: In what case the default view would be used ? Is it possible to land on this case ?

Comment (vbochetRTE) [reply src/utils/view-utils.js:21]: Default view is still used if you access the HMI without specifying the date parameter in the URL.

File: src/utils/date-time-utils.js (hunk 1 above)
Comment (vbochetRTE) [reply src/utils/date-time-utils.js:58]: By default, methods from the Date class in JavaScript return date and month as single digit if the value is less than 10. The `twoDigits()` function is used to add a "0" in such cases.
Moreover, the `getMonth()` method of JS' Date class counts from 0 (January = 0, February = 1, ...) which is not how we display months in ISO dates. The `twoDigitsMonth()` function is used to transform a JS month to a human-readable value representing that month.

Comment (vbochetRTE) [reply src/utils/date-time-utils.js:58]: If you have any proposal for functions' naming, feel free ;-)

**Review by TheoPascoli — APPROVED** [review]

**Review by oualid-aloui-rte — CHANGES_REQUESTED** [review]

**Review by schwitzguebel — APPROVED** [review]

**Pull request merged**